### PR TITLE
Fix ConnectDialog copy

### PR DESCRIPTION
**Pull request by angusjfw**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: frontend/src/pages/wifiPage/connectDialog/ConnectDialog.tsx [hunk 1]
```diff
@@ -105,8 +105,8 @@ export default ({
           <span className={styles.error}>
             {disconnectedFromAp ? (
               <>
-                Your computer has disconnected from the wifi network
-                'pi-top'. Please reconnect to it.
```
Comment (olivierwilkinson) [inline frontend/src/pages/wifiPage/connectDialog/ConnectDialog.tsx:109]: I think if the ssid above is using green instead of inverted commas then this should as well. Or we could say 'disconnected from the pi-top's WiFi hotspot' instead?
